api/pkg/dataflow/service.go

```diff
@@ -28,12 +28,19 @@ import (
 	dataflow "github.com/opensds/multi-cloud/dataflow/proto"
 	s3 "github.com/opensds/multi-cloud/s3/proto"
 	log "github.com/sirupsen/logrus"
+	"os"
 )
 
 const (
-	backendService  = "soda.multicloud.v1.backend"
-	s3Service       = "soda.multicloud.v1.s3"
-	dataflowService = "soda.multicloud.v1.dataflow"
+	MICRO_ENVIRONMENT = "MICRO_ENVIRONMENT"
```

Comment:
Suggestion: Many service names looks to be reused, can think of moving to common file in future